### ISSUE TITLE
Install errors

### ISSUE THREAD

**Issue by mariohmol** [evanshortiss/obd-parser-cli#1]
Hi!! Thanks for this project!!

When I try to install I get this errors, any ideas?

What nodejs version you are using?

thanks

node-pre-gyp http Pre-built binary not available for your system, looked for https://github.com/voodootikigod/node-serialport/releases/download/2.1.2/serialport-v2.1.2-node-v79-darwin-x64.tar.gz (falling back to source compile with node-gyp) 
node-pre-gyp verb command build [ 'rebuild' ]
node-pre-gyp http Connection closed while downloading tarball file (falling back to source compile with node-gyp) 
node-pre-gyp verb command build [ 'rebuild' ]
  CXX(target) Release/obj.target/serialport/src/serialport.o
  CXX(target) Release/obj.target/serialport/src/serialport.o

**Comment by mariohmol** [comment]
I could make it install on node 9 but i get this:

🚔  OBD CLI 🚘


#
# Fatal error in ../deps/v8/src/api.cc, line 1249
# Check failed: !value_obj->IsJSReceiver() || value_obj->IsTemplateInfo().
#
Illegal instruction: 4

**Comment by evanshortiss** [comment]
@mariohmol I was using this with Node.js 4 or 6 I think, but haven't tested it in a while. It appears to be an issue in compilation of the serialport module that this one depends on. 

I wonder if the [serialport dep here](https://github.com/evanshortiss/obd-parser-serial-connection/blob/master/package.json) needs to be updated. My guess is it's way out of date since it's 2.x

**Comment by evanshortiss** [comment]
TLDR; `obd-parser-serial-connection` module needs to be updated. I currently don't have access to hardware for testing, but am happy to accept a PR to `obd-parser-serial-connection` and this module to fix it.

**Comment by evanshortiss** [comment]
@mariohmol try the dev branch via `npm i -g evanshortiss/obd-parser-cli` - it might work since I updated the serialport version, but as I said, I can't test it right now.

EDIT: the dev branch will not work since it contains uncompiled assets. Please see my new command below.

**Comment by fklement** [comment]
@evanshortiss The installation via the `dev` branch does not work for me either.

```cli
> npm install evanshortiss/obd-parser-cli#dev -g               
npm ERR! code ENOENT
npm ERR! syscall chmod
npm ERR! path /Users/fklement/.nvm/versions/node/v10.15.3/lib/node_modules/obd-parser-cli/bin/index.js
npm ERR! errno -2
npm ERR! enoent ENOENT: no such file or directory, chmod '/Users/fklement/.nvm/versions/node/v10.15.3/lib/node_modules/obd-parser-cli/bin/index.js'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/fklement/.npm/_logs/2020-07-11T13_30_37_073Z-debug.log   
```

**Comment by evanshortiss** [comment]
@fklement please try `npm i -g obd-parser-cli@0.3.0-beta.0`. This installed for me with Node.js v14. I have not tested on with an real connection though as I currently don't have access to one.

**Comment by fklement** [comment]
@evanshortiss Nice that worked out fine!
I'm going to test it through a real connection and let you know.

**Comment by evanshortiss** [comment]
@fklement I appreciate it! The main issue is that the `serialport` module API/interface could have changes, but I hope not.
